spec.html

```diff
@@ -302,33 +305,62 @@ <h1>Runtime Semantics: ClassElementEvaluation</h1>
 
   </emu-clause>
 
+  <!-- es6num="14.5.15" -->
+  <emu-clause id="sec-runtime-semantics-bindingclassdeclarationevaluation">
+    <h1>Runtime Semantics: BindingClassDeclarationEvaluation</h1>
+    <emu-grammar>ClassDeclaration : <ins>DecoratorList?</ins> `class` BindingIdentifier ClassTail</emu-grammar>
+    <emu-alg>
+      1. <ins>If |DecoratorList_opt| is present, let _decorators_ be the result of performing DecoratorListEvaluation of |DecoratorList|.</ins>
```

Comment:
All this section has just been unindented. The only change is here (`DecoratorList` -> `DecoratorList_opt`)